### PR TITLE
[SP-5183] Add OTT PM support

### PR DESCRIPTION
**Pull request by Suryakantj**
- Now we have dedicated 'privacy manager' for OTT environment.

- OTT PM will act as first layer message

### CODE REVIEW

File: cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLib.java [hunk 1]
```diff
@@ -599,7 +602,8 @@ String pmUrl(String privacyManagerId, String pmTab) {
         params.add("pmTab="+pmTab);
         params.add("site_id="+ propertyId);
         if (consentUUID != null) params.add("consentUUID=" + consentUUID);
-        return PM_BASE_URL + "?" + TextUtils.join("&", params);
+        String PM_URL = isOTT ? OTT_PM_BASE_URL : PM_BASE_URL;
+        return PM_URL + "?" + TextUtils.join("&", params);
```
Comment (andresilveirah) [inline cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLib.java:606]: It'd be cool to test that if the `isOTT` flag is set, the WebView gets loaded with the correct URL.

Comment (Suryakantj) [reply cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLib.java:606]: @andresilveirah I have added unit test in ConsentLibBuilder class to check isOTT flag set

Comment (andresilveirah) [reply cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLib.java:606]: @Suryakantj right, you added a spec for the setter. I'm referring to the actual logic of changing from one PM url to another based on that flag.

Comment (Suryakantj) [reply cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLib.java:606]: @andresilveirah As suggested I have added unit to check correct pmURL.

File: cmplibrary/src/test/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLibTest.java [hunk 2]
```diff
@@ -205,4 +206,11 @@ public void customConsentTo() throws ConsentLibException, JSONException {
         assertEquals("foo_category", requestParams.getJSONArray("categories").get(0));
         assertEquals("foo_legIntCategory", requestParams.getJSONArray("legIntCategories").get(0));
     }
+
```
Comment (andresilveirah) [inline cmplibrary/src/test/java/com/sourcepoint/gdpr_cmplibrary/GDPRConsentLibTest.java:209]: nice!

**Review by andresilveirah — APPROVED** [review]

**Pull request merged**